source/jormungandr/jormungandr/interfaces/v1/serializer/api.py

```diff
@@ -190,23 +189,14 @@ class JourneysSerializer(PbNestedSerializer):
     feed_publishers = FeedPublisherSerializer(many=True, display_none=True)
     links = MethodField(schema_type=lambda: LinkSchema(), many=True)
     context = MethodField(schema_type=lambda: ContextSerializer(), display_none=False, many=True)
-    notes = MethodField(schema_type=lambda: NoteSerializer(), display_none=False, many=True)
-    exceptions = MethodField(schema_type=lambda: ExceptionSerializer(), display_none=False, many=True)
+    #TODO: We need to add add descriptions for attributes notes and exceptions to be used by SDK
```

Comment:
actually I think you can write those (not tested, it might need to be adapted):

```python
class DescribedField(LambdaField):
"""This class does not output anything, it's here only for description purpose (for field added outside of serpy, but that we want to describe in swagger"""
    def __init__(self, method, **kwargs):
        # the field returns always None and None are not displayed, so nothing is very displayed
        super(DescribedField, self).__init__(method=lambda *args: None, display_none=False, **kwargs)

notes = DescribedField(schema_type=NoteSerializer(), many=True)
exceptions = DescribedField(schema_type=ExceptionSerializer(), many=True)
```